### DISCUSSION TITLE
Not able to run application locally

### DISCUSSION THREAD

**Discussion by coderj001** [danny-avila/LibreChat#3937] [Troubleshooting]
### What is your question?

![image](https://github.com/user-attachments/assets/8260fe45-d567-4fcc-8cac-7962b27aa63d)

I have follow all instruction and run `docker compose up` but not able make it run properly.

### More Details

I'm running arch linux, kernal version 6.6.48-1-lts, Docker version 27.2.0, build 3ab4256958 and Docker Compose version 2.29.2

### What is the main subject of your question?

Installation

### Screenshots

![image](https://github.com/user-attachments/assets/b0dde500-a5bf-4a13-a5cb-c33975cc1082)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Take a look at this discussion: https://github.com/danny-avila/LibreChat/discussions/3647#discussioncomment-10342994
